src/backend/opencl/kernel/bilateral.hpp

```diff
@@ -49,7 +49,8 @@ void bilateral(Param out, const Param in, float s_sigma, float c_sigma)
 
     std::call_once( compileFlags[device], [device] () {
             bool use_native_exp = (getActivePlatform() != AFCL_PLATFORM_POCL
-                                && getActivePlatform() != AFCL_PLATFORM_APPLE);
+                                && getActivePlatform() != AFCL_PLATFORM_APPLE
+                                && getActivePlatform() != AFCL_PLATFORM_AMD);
```

Comment:
@pavanky No doubt supported in the past against their dead backend (AMD APPSDK). Not sure about amdgpu-pro but anyway it's either that or another product that is soley for AMD products that also passes the existing AMD platform mapping. I added platform support for that and then decided it was better not to. Besides, why encourage a fast crap accuracy exp over a fast high accuracy exp.